deploy/storage/base_bigtable.yaml

```diff
@@ -0,0 +1,7 @@
+project: datcom-store
+instance: prophet-cache
+tables:
+  - frequent_2022_10_25_01_16_38
+  - ipcc_2022_10_03_12_40_31
+  - biomedical_2022_10_21_14_26_25
+  - infrequent_2022_10_24_11_46_31
```

Comment:
There are references to the previous version of the file, esp from GC script: https://source.corp.google.com/search?q=bigtable_import_groups.version

Reply:
Also, if we're changing the format, some parsing in the script might need to change.